File: Begginers/Python/tipo_de_combustivel_1134.py
alcool = 0
gasolina = 0
diesel = 0

while True:
      codigo = int(input())
      if 1 <= codigo <= 4:
         if codigo == 1:
            alcool += 1
         elif codigo == 2:
            gasolina += 1
         elif codigo == 3:
            diesel += 1
         elif codigo == 4:
              break
       
print(f"MUITO OBRIGADO")
print(f"Alcool: {alcool}")
print(f"Gasolina: {gasolina}")
print(f"Diesel: {diesel}")

File: Begginers/Java/tipo_de_combustivel_1134.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
       public static void main(String[] args) throws IOException {
              Scanner sc = new Scanner(System.in);

              int x, contador_gasolina, contador_alcool, contador_diesel;
              contador_gasolina = 0;
              contador_alcool = 0;
              contador_diesel = 0;

              while (true) {
                     x = sc.nextInt();

                     if (x == 4) {
                            break;
                     }
                     else if (x == 1) {
                            contador_alcool += 1;
                     }
                     else if (x == 2) {
                            contador_gasolina += 1;
                     }
                     else if (x == 3) {
                            contador_diesel += 1;
                     }
              }
              
              System.out.println("MUITO OBRIGADO");
              System.out.println("Alcool: " + contador_alcool);
              System.out.println("Gasolina: " + contador_gasolina);
              System.out.println("Diesel: " + contador_diesel);
              
              sc.close();

       }
}
